### DISCUSSION TITLE
Give Github Sponsors Credit as a gift.

### DISCUSSION THREAD

**Discussion by N5-nfive** [community/community#36699] [Other Feature Feedback, Questions, & Ideas]
Dear community,

I would like make a fellow developer a birthday present.   
Is it possible to somehow give "credit" to their GitHub account, so they can use it to sponsor projects they like?

As far as I can tell, it is only possible to pay with PayPal and Credit Card, so it is not even an option to create a fresh account and top it up with e.g. Amazon gift cards or whatever. 

Does anyone have an idea, how to hand out GitHub Sponsors money as a gift?

**Comment by ossanna16** [comment]
Hey there @N5-nfive and thank you for asking a good question! I'm glad you found our community 🙂
